apps/users/models/users.py

```diff
@@ -34,5 +34,23 @@ def status_value(self) -> Optional[str]:
     def status_value(self, value: Optional[str]) -> None:
         self._status_value = value
 
+    @property
+    def role(self) -> str:
+        if self.is_superuser:
+            return "admin"
+        if self.is_staff:
+            return "staff"
+        return "user"
+
+    @property
+    def status(self) -> str:
+        if self.is_active:
+            return "active"
+        if self.pk is None:
+            return "inactive"
+        if self.withdrawals.exists():
+            return "withdrew"
+        return "inactive"
+
```

Comment:
이거 조건이 잘못된 것 같습니다
```suggestion
    @property
    def status(self) -> str:
        if self.is_active:
            return "active"
        if self.withdrawals.exists():
            return "withdrew"
        return "inactive"
```

이렇게되어야하지않을까요?

유저의 is_active가 True이면 활성화된 유저,
is_active가 false이고 탈퇴내역이 존재하지않으면 비활성화만 된 유저
is_active가 false이고 탈퇴내역이 존재하면 회원탈퇴가 진행중인 유저